topics/persistence.md

```diff
@@ -34,6 +37,9 @@ AOF advantages
 * Using AOF Redis is much more durable: you can have different fsync policies: no fsync at all, fsync every second, fsync at every query. With the default policy of fsync every second write performances are still great (fsync is performed using a background thread and the main thread will try hard to perform writes when no fsync is in progress.) but you can only lose one second worth of writes.
 * The AOF log is an append only log, so there are no seeks, nor corruption problems if there is a power outage. Even if the log ends with an half-written command for some reason (disk full or other reasons) the redis-check-aof tool is able to fix it easily.
 * Redis is able to automatically rewrite the AOF in background when it gets too big. The rewrite is completely safe as while Redis continues appending to the old file, a completely new one is produced with the minimal set of operations needed to create the current data set, and once this second file is ready Redis switches the two and starts appending to the new one.
+
+  Since Redis 7.0.0, the rewrite is also completely safe. When an AOF rewrite is scheduled, Redis parent process opens a new INCR AOF file to continue writing. The child process will execute rewrite logic and generates a new BASE AOF. Redis will use a temporary manifest file to track the newly generated BASE file and INCR file. When they are ready, Redis will perform an atomic replacement operation to make this temporary manifest file take effect.At the same time, in order to avoid the problem of creating many INCR files due to the frequent failure and retries of AOF rewrite. Redis introduced an AOF rewrite limiting mechanism to ensure that AOF rewrite automatically retries at a slower and slower speed.
```

Comment:
```suggestion
  Since Redis 7.0.0, the rewrite is also completely safe. When an AOF rewrite is scheduled, The Redis parent process opens a new INCR AOF file to continue writing. The child process will execute rewrite logic and generates a new BASE AOF. Redis will use a temporary manifest file to track the newly generated BASE file and INCR file. When they are ready, Redis will perform an atomic replacement operation to make this temporary manifest file take effect. At the same time, in order to avoid the problem of creating many INCR files in case of repeated failures and retries of an AOF rewrite, Redis introduces an AOF rewrite limiting mechanism to ensure that AOF rewrite automatically retries at a slower and slower speed.
```

Reply:
OK